twifbeddit-app/backend/src/posts.js

```diff
@@ -24,13 +24,34 @@ exports.POST = async (_, event) => {
     }
 };
 
-// TODO: pagination
+// TODO: Test this. Get posts that a specific user is following, sorts by most recent posts.
 exports.GET = async (_, event) => {
     if (event.queryStringParameters != null)
         console.error("Parameters currently ignored");
     const username = await get_user_from_header(event.headers);
     const user = await User.findOne({username: username});
-    var posts = await Post.find().where('topic').in(user.following).exec();
+    var posts = await Post.find().where('topic').in(user.followed_topics).limit(8).sort({ createdAt: -1}).exec();
```

Comment:
Why `limit(8)`? Does this just show 8 posts? Might as well just send all, not like anyone is using this anyway...

Reply:
If they have a for loop without a known limit, it could get messy. We may or may not want pagination in the future. This can just be temporary.